src/AppBundle/Resources/views/Search/card-front.html.twig

```diff
@@ -19,6 +19,9 @@
 				{% endif %}
 				{% include 'AppBundle:Search:card-illustrator.html.twig' %}
 				{% include 'AppBundle:Search:card-pack.html.twig' %}
+				{% if card.erratum %}
+					<div style="margin-top: 2rem;"><i><b>Erratum:</b> {{card.erratum}}</i></div>
```

Comment:
I think we'll probably want the colon to be outside the bold tag to match how card actions are written (for example `<b>Response</b>:`)